### PR TITLE
[Submission] Powell

### PR DESCRIPTION
**Pull request by jmbpowell**
**Jessica Powell**

Dear mdx workshop committee,

I request a review for the following MDX submission:

### **New applications of music separation for the music industry**  

**Author(s):**  

**Category ***

* [ ] `POSTERS` _(+Abstract: 250 Words+one page pdf)_
  > As a participant of the MDX challenge, please know that your poster would be automatically accepted after a minimal prescreening. Hence, don't forget to provide us with the link to your team in the AIcrowd website.
  > You are also welcome to submit us some unrelated abstract, in case your interest would be in presenting some method or idea you would just like to discuss with the participants passing by.  
  > => You *don't* need to include your poster in the PR. Just be ready to have it for D-day and show it to the audience through screen sharing.
* [ ] `LONG TALK`
  > A long talk will last about 20-30min, including questions. This can be about some recent paper you published, or may also present some original research and investigations you did. If you pick this category, please provide us with an extended abstract of the talk, or directly with the slides if you have them available, even in an unfinished state, at your convenience. 
* [x] `DISCUSSIONS`
  > This category is intended to stimulate fruitful discussions and collaborations. If you pick it, it means you are willing to chair some open discussion with the participants, mainly around one particular aspect of music demixing research. For that purpose, you should provide a minimal abstract as well as a list of a few questions you would like the participants to discuss about. Don't hesitate to propose ideas, We will possibly contact you in case we identify several similar topics that were proposed!

**Preferred Timezone***

* [x] A 11:00 PM (PDT) to 9:00 AM PDT
* [ ] B 11:00 AM (PDT) to 9:00 PM PDT

  > The preferred timezone will help us to organize the day schedule. You should select at least one, but can perfectly select both if you don't care. We will organize the programme so that to maximally fit your preferences.
---

**ORGANISATION COMMITTEE (do not fill out)**

* [x] Editor acknowledgment
* [x] Reviewer 1 
* [ ] Reviewer 2
* [x] Review 1 decision [accept/reject]
* [ ] Review 2 decision [accept/reject]
* [x] Editor decision [accept/reject]

### CODE REVIEW

**Comment by faroit** [comment]
@jmbpowell Thanks for the submission 👏

File: paper.md [hunk 1]
```diff
@@ -1,139 +1,33 @@
...
-# Paper structure depending on the category
+Moderated by Antoine Liutkus (INRIA), our panel will bring together a researcher (James Clarke, Abbey Road Studios), a start-up CEO (Jessica Powell, Audioshake, winner of the 2021 Demixing Challenge), and someone from the music industry (Mike Daly, Executive Director, A&R for Disney Music Group).
 
-## Posters
+The goal is to provide the audience with more insight into what it’s like to apply source separation to business use cases, and also hear from the industry perspective about how they think about AI, and where they think it can help.
 
-In case of a poster that presents a MDX submission, please atinclude a _method_ section,
-where you describe your system. It would be nice to have both explanations and at least
-one figure of the architecture / model / whatever you see fit.
+# Potential Questions
```
Comment (aliutkus) [inline paper.md:27]: Potential questions that I feel could be interesting:
* Source separation allows to recover separated stems. Is this perceived as a dangerous technology by catalogue owners ? What is the appropriate way to present the technology in this context ?
* There has been many attempts at developing "active listening" offers where users would interact with the music content, but none has received wide acceptance so far. Do you think there is a future for a wide audience use case of source separation ? Which one ?
* An obvious application of source separation is for music creation purposes where amateur musicians could modify or mix stems from existing songs. Is the music industry equipped (legally/psychologically) for this move ?
* Audio engineering is often perceived as the art of producing a mixture from stems. Source separation may be perceived as a way to deconstruct mixtures, to generate new ones on the fly. Is this considered a dangerous technology from a sound engineering perspective ? What would be here again the correct way to present the technology to professionals ?
* Going even further than universal separation, let's assume that music demixing works fine. What would be the next holy grail ? What should we, as audio researchers, focus on to create impact for you ?

**Review by aliutkus — COMMENTED** [review]
It will be a great discussion

**Comment by aliutkus** [comment]
To all participants:
please note that you should update your submission by November 6th 23:00 UTC

Thank you very much

File: paper.md (hunk 1 above)
Comment (faroit) [reply paper.md:27]: @aliutkus these are great additions. @jmbpowell we will add them to the proposal

**Comment by faroit** [comment]
👏 The submission has been published on midspace (the ISMIR conference platform). The submissions will also be hosted on the workshop website tomorrow. 

The schedule can be seen there or [here](https://docs.google.com/spreadsheets/d/1PaksFcHWUfNPMg-XloA3CJtrvb9uMU0erCFVyvFi9B4/edit?usp=sharing). Please let us know if there is anything missing. Otherwise we see each other on Friday/Saturday!

**Pull request closed without merging**